File: Iniciante/URI 1046.rb
line = gets.split(' ')
hInicial = line[0].to_i
hFinal = line[1].to_i

if hInicial > hFinal
    puts "O JOGO DUROU #{24 - (hInicial - hFinal)} HORA(S)"
elsif hFinal > hInicial
    puts "O JOGO DUROU #{hFinal - hInicial} HORA(S)"
else
    puts "O JOGO DUROU 24 HORA(S)"
end

File: Iniciante/URI 1046.pas
var
	hInicial : int64;
	hFinal : int64;

begin
    readLn(hInicial, hFinal);
	if (hInicial > hFinal) then
		writeln('O JOGO DUROU ', (24 - (hInicial - hFinal)), ' HORA(S)')
	else if (hFinal > hInicial) then
		writeln('O JOGO DUROU ',  (hFinal - hInicial), ' HORA(S)')
	else
		writeln('O JOGO DUROU 24 HORA(S)')
end.